test/test_sassc.rb

```diff
@@ -0,0 +1,364 @@
+require 'sprockets_test'
```

Comment:
Yep! Btw, you can use `SassC` script types like `SassC::Script::String` and `Sass` script types like `SassC::Script::String` interchangeably with `SassC` in case that makes things easier.